test/inductor/test_torchinductor_opinfo.py

```diff
@@ -215,7 +215,7 @@ def format_op(op):
     "complex": {f16},
     "resize_": {b8, f16, f32, f64, i32, i64},
     "resize_as_": {b8, f16, f32, f64, i32, i64},
-    "histc": {f16},
+    "histc": {f16, i32, i64},
```

Comment:
Should be able to remove these skips now?

Reply:
`int32` and `int64` input tensors are still expected to fail for `device="cpu"`, so this should be kept.

Reply:
These shouldn't be run on CPU, as the OpInfo specifies the types correctly:
```python
    OpInfo('histc',
           dtypes=floating_types_and(torch.bfloat16, torch.float16),
           dtypesIfCUDA=floating_types_and(torch.int8, torch.int16, torch.int32, torch.int64),
```